src/pages/expenses/documents/Documents.tsx

```diff
@@ -51,12 +53,26 @@ export default function Documents() {
 
   const queryClient = useQueryClient();
 
+  const actions = useActions();
+
   const invalidateCache = () => {
     queryClient.invalidateQueries(route('/api/v1/expenses/:id', { id }));
   };
 
   return (
-    <Default title={documentTitle} breadcrumbs={pages}>
+    <Default
+      title={documentTitle}
+      breadcrumbs={pages}
+      navigationTopRight={
+        expense && (
+          <ResourceActions
+            resource={expense}
+            label={t('more_actions')}
+            actions={actions}
+          />
+        )
+      }
+    >
```

Comment:
@beganovich After reviewing this section of the code, I noticed that it could potentially be improved by creating separate components, `Expense.tsx` and `RecurringExpense.tsx`, to handle the tabs on the edit pages for these entities.

I believe we have implemented a similar approach wherever we have tabs on edit pages. This will ensure that we don't miss the `Save` button in the navigation bar on `Documents` tab for example and avoid duplicating the setting of the `navigationTopRight` property.

Although I didn't include this implementation in the current PR since it's unrelated, I wanted to recommend it.

@turbo124 If we consider the absence of the `Save` button on the `Documents` tab for those Edit pages as a bug, we could create a ticket to address it and make some minor improvements to the code implementation.

Please let me know your thoughts on this suggestion.

Reply:
@Civolilah yes, please create a ticket for this, save button should be present.